src/components/table/partial-styles/tabulator-arrow.scss

```diff
@@ -25,7 +25,7 @@
         bottom: 0;
     }
     &:before {
-        z-index: 1;
+        z-index: $limel-table--tabulator-arrow--before--z-index;
```

Comment:
I've double checked now. There is no need for `z-indexes` for `:before` and `:after` here for the arrows, as the browser renders them already correctly. Checked with latest chrome and firefox. Let's remove them.

Reply:
⚡